src/navigations/HomeBottomTabs.js

```diff
@@ -138,7 +142,7 @@ function HomeBottomTabs({navigation}) {
     PushNotification.createChannel(
       {
         channelId: 'bettersosialid', // (required)
-        channelName: 'bettersosial-chat', // (required)
+        channelName: 'New Messages & Comments', // (required)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The channel name has been changed from 'bettersosial-chat' to 'New Messages & Comments'. Make sure this change is reflected wherever the channel name is used in your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->